vocabularies/UI.xml

```diff
@@ -1372,7 +1372,13 @@ The trend is
 
       <Term Name="TextArrangement" Type="UI.TextArrangementType" Nullable="false" AppliesTo="Annotation EntityType">
         <Annotation Term="Core.Description" String="Describes the arrangement of a code or ID value and its text" />
-        <Annotation Term="Core.LongDescription" String="If used for a single property the Common.Text annotation is annotated" />
+        <Annotation Term="Core.LongDescription">
+          <String>This term annotates one of the following:&lt;br&gt;
+          (1) a [`Common.Text`](Common.md#Text) annotation of the code or ID property whose value is the text&lt;br&gt;
```

Comment:
imo is "code or ID property whose value is the text" misleading (maybe you mean with "whose" the annotation?). I would suggest to use only the first part "(1) a [`Common.Text`](Common.md#Text) annotation of a code or an ID property" - what do you think?